.gitlab-ci.yml

```diff
@@ -165,7 +174,7 @@ test-linux-stable-int:
         &> ${CI_COMMIT_SHORT_SHA}_int_failure.log
     - sccache -s
   after_script:
-    - awk '/FAILED/,0' ${CI_COMMIT_SHORT_SHA}_int_failure.log
+    - awk '/FAILED|^error\[|^warning:/,0' ${CI_COMMIT_SHORT_SHA}_int_failure.log
```

Comment:
Does this makes the integration test fail when we find a `warning`?
In general the integration test is compiling the same code as `test-linux-stable`, so we should not get any different warnings.

Reply:
no, it doesn't. It just collects the log starting from the first word mentioned there. Ok, I'll remove the `warning` from there.